modules/core/src/test/scala/fs2/kafka/KafkaConsumerSpec.scala

```diff
@@ -707,6 +793,65 @@ final class KafkaConsumerSpec extends BaseKafkaSpec {
       }
     }
 
+    it("should stream assignment updates to listeners when using CooperativeStickyAssignor") {
```

Comment:
This is a copy of the existing `"should stream assignment updates to listeners"` test, adapted to use the custom assignor, along with the resulting asserts.